packages/ui/src/theme/overwrites/components/textarea.ts

```diff
@@ -0,0 +1,34 @@
+export const Textarea = {
+  baseStyle: {
+    borderColor: 'gray.300',
+  },
+  variants: {
+    borderColor: 'gray.300',
+    outline: {
+      borderRadius: '6px',
+      _focus: {
+        boxShadow: 'outline',
+      },
+      _disabled: {
+        borderColor: 'gray.300',
+      },
+      _hover: {
+        borderColor: 'gray.400',
+      },
+    },
+  },
+  sizes: {
+    sm: {
+      px: '12px',
+      py: '6px',
+    },
+    md: {
+      px: '16px',
+      py: '8px',
+    },
+    lg: {
+      px: '16px',
+      py: '10px',
+    },
+  },
+};
```

Comment:
So, there's a problem with this file, styles aren't applied correctly and there are typing problems. I've checked it locally and found a solution for that. You should wrap those properties in a field object. For example: 
```typescript
baseStyle: {
    field: {
      borderColor: 'gray.300',
    }
  }
```
You can also take a look at how was it implemented in radio overwrites because it's very similar.